segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -137,6 +144,12 @@ public void shutdown() {
                 segmentStatsFactory.close();
             }
 
+            if (this.serviceConfig.isSecureZK()) {
+                System.clearProperty(SECURE_CLIENT);
```

Comment:
Perhaps, clearing these properties may also be encapsulated in a helper method.

Reply:
Fixed.